test/integration/build/aws.tf

```diff
@@ -1411,6 +1413,33 @@ resource "aws_eks_cluster" "aws_eks_cluster" {
   }
 }
 
+resource "aws_subnet" "eks_subnet-3" {
+  count             = var.aws_enable_creation
+  vpc_id            = aws_vpc.eks_vpc[0].id
+  availability_zone = "${var.aws_region}b"
+  cidr_block        = "10.0.50.0/20"
+
+  depends_on = [aws_internet_gateway.igw]
+
+  tags = {
+    Name = var.aws_eks_subnet_name_1
+  }
+}
+
+resource "aws_db_subnet_group" "aws_db_subnet_group_1" {
```

Comment:
Maybe along the same lines as above... `rds_subnet_group`

Reply:
Updated in recent commit